cmd/extproc/main.go

```diff
@@ -1,5 +1,5 @@
-// Copyright Envoy AI Gateway Authors
-// SPDX-License-Identifier: Apache-2.0
+// Copyright Envoy AI Gateway Authors.
+// SPDX-License-Identifier: Apache-2.0.
```

Comment:
no you shouldn't modify the generated license header

Reply:
it looks we need a more advanced regex :D

Reply:
updated